src/power_divergence.jl

```diff
@@ -299,6 +367,11 @@ ChisqTest{T<:Integer,U<:AbstractFloat}(x::AbstractVector{T}, theta0::Vector{U} =
     PowerDivergenceTest(reshape(x, length(x), 1), lambda=1.0, theta0=theta0)
 
 #MultinomialLRT
+"""
+    MultinomialLRT(x [,y] [,theta0])
+
+Convenience function for power divergence test with ``λ=0``.
```

Comment:
Explain more what this test means so that it can be used in isolation. Link to `PowerDivergenceTest`.